ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -152,7 +152,7 @@ check_mechanism_against_processing(struct pkcs11_session *session,
 static uint8_t *pkcs11_object_default_boolprop(uint32_t attribute)
 {
 	static const uint8_t bool_true = 1;
-	static const uint8_t bool_false;
+	static const uint8_t bool_false = 0;
```

Comment:
static variables are expected to be 0 initialized by construction.

Reply:
Exception for what, assigning explicitly to `0`?

Reply:
Aha, sorry I thought this would result in yet another exception. This clarification sounds good to me.